### ISSUE TITLE
Fields are displayed as raw text in activity stream

### ISSUE THREAD

**Issue by yvanzo** [fulstech/markin#49]
Fields with Markin Renderer are displayed as raw text in Activity Stream gadget (Full view), whereas fields with Wiki Style Renderer were displayed normally.

**Comment by hozn** [comment]
Yeah, we're noticing this too.  It makes the mentions etc. pretty useless here.  Not sure if this is quite a deal-breaker, but it definitely affects usability for us.
